src/Capsule/Collection.php

```diff
@@ -221,7 +221,8 @@ private function prepareQuery(string $method, $filters = null, $data = null, arr
         );
 
         $event = new QueryEvent($query);
-        if (class_exists(LegacyEventDispatcherProxy::class)) {
+        // The Event class has been removed in Symfony 5, so its absence is used as a trigger to stop using the LegacyEventDispatcherProxy.
+        if (! class_exists(\Symfony\Component\EventDispatcher\Event::class) || class_exists(LegacyEventDispatcherProxy::class)) {
```

Comment:
This condition is often repeated. Is it possible to encapsulate it in an external static function?